package.json

```diff
@@ -40,15 +40,20 @@
             "jspm": "src/configs/jspm.config.js"
         },
         "dependencies": {
+            "@types/google-images": "npm:@types/google-images@^2.1.0",
```

Comment:
No need to have a types package as a dependency in JSPM. It used in development only. It should be moved to  `devDependency` of `npm`.